salt/_modules/metalk8s_checks.py

```diff
@@ -434,6 +442,23 @@ def route_exists(destination, raises=True):
     return error or True
 
 
+def containerd_filesystem(folder=None, raises=True, **kwargs):
+    error = None
+    folder = folder or "/var/lib/containerd"
+    while not os.path.exists(folder):
+        folder = os.path.dirname(folder)
+    try:
+        infos = __salt__["xfs.info"](folder)
+        ftype = infos["naming"]["ftype"]
+        if ftype != "1":
+            error = f"XFS filesystem has ftype={ftype} expected 1"
+    except CommandExecutionError:
+        error = f"{folder} is not on an XFS"
```

Comment:
It's not an error just we want to check if it's xfs then `ftype` should be set to `1` but it could be something else than xfs it's ok also. 
Maybe we should enforce ext4 but for the moment I think if it's not XFS then it's ok **but** IMHO we should really check if it's xfs and not rely on "xfs.info" failure